src/data-pipeline/spark-etl/src/main/java/software/aws/solution/clickstream/Cleaner.java

```diff
@@ -41,11 +41,13 @@
 import static java.util.Objects.requireNonNull;
 import static org.apache.spark.sql.functions.*;
 import static org.apache.spark.sql.types.DataTypes.StringType;
+import static software.aws.solution.clickstream.ContextUtil.*;
```

Comment:
use certain class name instead of using wildcard?

Reply:
also add a checksytle rule to check it

Reply:
changed.